beacon-node-oapi.yaml

```diff
@@ -21,7 +21,7 @@ info:
 
     Note that it is possible for a field to be added to an endpoint's data or metadata without an increase in the version number.
 
-  version: "Dev - Eth2Spec v1.1.0"
```

Comment:
Is the name `Eth2Spec` offical? Searching for it brought me to this outdated page: https://pypi.org/project/eth2spec. And https://github.com/ethereum/consensus-specs is named `Ethereum Proof-of-Stake Consensus Specifications`. Maybe we should rename it?

Reply:
fair point, i can probably change the name now :).

Reply:
The drop-down is pretty short so might still be some abbreviation